demonstrations/tutorial_learning_dynamics_incoherently.py

```diff
@@ -0,0 +1,306 @@
+r"""How to learn quantum dynamics incoherently
+==========================================
+
+If we have an unknown quantum process that takes a quantum state as input and outputs
+another state, how can we recreate it or simulate it in a quantum circuit? How do we create a
+model circuit that reproduces the target quantum process? One approach is to learn the
+dynamics of this process incoherently [#Jerbi]_. In simple terms, this
+consists of two steps:
```

Comment:
(optional) I wonder if this simplifies the intro a bit:
```suggestion
If we have an unknown quantum process that takes a quantum state as input and outputs
another state, how can we recreate it or simulate it in a quantum circuit? One approach is to learn the
dynamics of this process incoherently [#Jerbi]_. In simple terms, this
consists of two steps:
```